reactor-core/src/test/java/reactor/core/publisher/LiftFunctionTest.java

```diff
@@ -37,10 +40,21 @@
 public class LiftFunctionTest {
 
 	Publisher<Integer> liftOperator;
+	String lifterName;
```

Comment:
encapsulation is not great in this class, and adding more nesting level is definitely not a good thing IMHO. can you try to rework that? I don't think we really need 1 `Base` class per type of publisher...

however, maybe `liftOperator` and `lifterName` (as well as assertion methods) would benefit from being moved to a _single_ base class for all the nested cases?

Reply:
done